src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx

```diff
@@ -12,37 +12,49 @@ import LimitedTextAreaInput from '../forminator/inputs/LimitedTextAreaInput';
 interface OwnProps {
   title: string;
   maxLength: number;
+  label?: string;
 }
 
 type Props = FCProps<OwnProps>;
 
-export function StrengthsOrWeaknesses({ title, maxLength, ...props }: Props) {
+export function StrengthsOrWeaknesses({ title, maxLength, label, ...props }: Props) {
   const lens = useFragmentLens();
   const condition = useCallback(
     (value: unknown[]) => {
-      return value ? value.length < maxLength : true;
```

Comment:
It's nice to revert this as value ? condition : true is much more readable than !value || condition;